### DISCUSSION TITLE
multi vector search terms

### DISCUSSION THREAD

**Discussion by snozwoz** [milvus-io/milvus#31851] [Q&A and General discussion]
i want to do a single vector search with multiple search terms. e.g.
* caffeine free drinks
* herbal tees
* energy drinks
Returned in a single search result. So if an article covers all 3x subjects it will be top of the results. 
It's part of a RAG-fusion project and trying to avoid multiple searches. 
i currently have it working in elastic search using multiple script_scores , which I am migrating away from. 
thanks in advance.

**Comment by xiaofan-luan** [comment]
simply put all the keywords in your search and embedding it see how it works

**Reply by snozwoz** [reply]
my example was a bit simplistic, the search terms are complete phrases, not keywords.

**Reply by xiaofan-luan** [reply]
sparse embedding could be something you can try and check how good it is.
I've seen another post about clustering everything. those sounds to be interesting topics to be discussed.
Feel free to link me at james.luan@zilliz.com and we can do further talk

**Comment by xiaofan-luan** [comment]
or if you have tags on all items, it's a filtering
